vuepress/docs/docs/stitching.md

```diff
@@ -0,0 +1,200 @@
+# Stitching
```

Comment:
By the way you need to add this in config.js for it to appear in the sidebar (maybe below federation?).

Reply:
Might be worth mentioning it in `tools.md` as well.

Reply:
Added in a3ffb92.